lib/cli.js

```diff
@@ -193,6 +187,10 @@ async function translateOptions({
         options.useEslintrc = eslintrc;
         options.extensions = ext;
         options.ignorePath = ignorePath;
+    } else {
+        if (ignorePattern) {
```

Comment:
Can we rearrange the conditionals in these blocks? It’s a bit hard to follow an else after a not equals. Maybe structure as “if configType is flat, then add ignorePatterns, else do the other stuff”?

Also, I don’t think you need to check for ignorePattern before adding ignorePatterns to options.

Reply:
Done, thanks for the suggestion!